cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/BufferingTable.java

```diff
@@ -496,9 +496,8 @@ private void putInternal(byte[] row, byte[][] columns, byte[][] values) {
   }
 
   /**
-   * NOTE: Depending on the use-case, calling this method may be much less
-   *       efficient than calling same method with columns as parameters because it may always require round trip to
-   *       persistent store
+   * NOTE: Depending on the use-case, calling this method may be much less efficient than calling same method
+   *       with columns as parameters because it may always require round trip to persistent store.
```

Comment:
ok. I wonder whether that can be changed now. I believe that Tephra's TransactionalTable does use row deletes. Belongs into a different PR, though. However, you should remove the "may" because it is certain that it requires a round trip.